ui_framework/doc_site/src/services/routes/routes.js

```diff
@@ -134,6 +137,10 @@ const components = [{
   name: 'Card',
   component: CardExample,
 }, {
+  name: 'Color Picker',
```

Comment:
Minor nit, can we change this to `ColorPicker` to be consistent with the component name?